stream-chat-android/src/main/java/com/getstream/sdk/chat/view/messageinput/MessageInputStyle.kt

```diff
@@ -10,24 +10,25 @@ import android.text.TextUtils
 import android.util.AttributeSet
 import androidx.annotation.ColorInt
 import androidx.annotation.DrawableRes
+import androidx.core.content.ContextCompat
 import androidx.core.graphics.drawable.DrawableCompat
 import com.getstream.sdk.chat.R
 import com.getstream.sdk.chat.style.TextStyle
-import com.getstream.sdk.chat.view.BaseStyle
+import com.getstream.sdk.chat.view.messages.AvatarStyle
 
 /**
  * Style for MessageInputStyle customization by xml attributes
  */
-internal class MessageInputStyle(context: Context, attrs: AttributeSet?) : BaseStyle() {
-
-    var inputText: TextStyle
-    val isShowAttachmentButton: Boolean
-    val inputButtonWidth: Int
-    val inputButtonHeight: Int
-    val attachmentButtonWidth: Int
-    val attachmentButtonHeight: Int
-    val attachmentCloseButtonBackground: Drawable
-    val inputSendAlsoToChannelTextColor: Int
+public class MessageInputStyle(private val context: Context, attrs: AttributeSet?) {
```

Comment:
Why does this class have to be `public`?

Reply:
It doesn't have to, good catch!